packages/test-config/jest-unit.config.common.ts

```diff
@@ -22,6 +22,7 @@ const config: Partial<Config.InitialProjectOptions> = {
             },
         ],
     },
+    transformIgnorePatterns: [],
```

Comment:
Are we sure that's not causing side-effects for existing tests?

Reply:
I'll make it more specific.

```
transformIgnorePatterns: ['/node_modules/(?!.*\\@noble/ed25519/)', '\\.pnp\\.[^\\/]+$'],
```